helpers/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .notification import Notification
```

Comment:
We generally avoid using relative imports as they can cause path issues, especially in a containerised environment. This is particularly relevant in our DAG context, where the Airflow configuration resides in a separate repository.